pkg/tests/mappings/mappings_test.bzl

```diff
@@ -126,12 +131,9 @@ def _test_pkg_files_contents():
     )
 
     # Used in the following tests
-    #
-    # Note that since the pkg_files rule is never actually used in anything
-    # other than this test, nonexistent_script can be included with no ill effects. :P
-    native.sh_binary(
+    fake_artifact(
```

Comment:
Might be worth explaining what `fake_artifact` replaces in the build in your commit message.

Reply:
Done